localstack/services/cloudformation/models/events.py

```diff
@@ -130,3 +131,69 @@ def wrap_in_lists(o, **kwargs):
                 "parameters": {"Name": "PhysicalResourceId"},
             },
         }
+
+
+class EventBusPolicy(GenericBaseModel):
+    def get_physical_resource_id(self, attribute=None, **kwargs):
+        return f"EventBusPolicy-{short_uid()}"
```

Comment:
I've implemented the version in _create like with the sqs queue policy for now because I want to go over all the other PhysicalResourceId generation cases before introducing a new mechanism. We have it on the refactoring list anyway :+1: